### ISSUE TITLE
bench setup backups : create a wrong crontab instruction  : Error: no such option: --site

### ISSUE THREAD

**Issue by FHenry** [frappe/bench#1238]
$ bench setup backups

Create a crontjob task  :   

`0 */6 * * * cd /home/erpnext/erpnext-XXX && /usr/local/bin/bench --site erpnextXXX.XXXX.XX backup --with-files --verbose >> /home/erpnext/erpnext-XXX/logs/backup.log 2>&1 # bench auto backups set for every 6 hours`

The command 
`/usr/local/bin/bench --site erpnextXXX.XXXX.XX backup --with-files --verbose `
return Error: no such option: --site

Solution should be update bench/bench/bench.py
```
def backups(self):

backup_command = f"cd {bench_dir} && {sys.argv[0]} --verbose --site all backup"

```
Should be 

`backup_command = f"cd {bench_dir} && {sys.argv[0]} --site {**"Site name"**} backup --with-files --verbose"`

**Comment by gavindsouza** [comment]
This seems to be an issue related to the recent changes introduced via #1208. Similar issue: #1223

**Comment by gavindsouza** [comment]
Should be fixed by #1243.
